dev/breeze/src/airflow_breeze/__init__.py

```diff
@@ -14,3 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+NAME = "Breeze2"
+VERSION = "0.0.1"
```

Comment:
When do we need to bump this? Maybe we should just set this to `2`.

Reply:
Yeah. Actually we do not have version yet. I have another PR when I switch it to 2.0.0 indeed (when I switch everyone to use the new Breeze